toolbox/log-purge/log-purge.cron

```diff
@@ -0,0 +1,11 @@
+# Cron job to clean up old files and empty directories from /usr/airflow/atd-airflow/logs
+# This will run nightly at 2 AM. The first command removes all files which have a modification
+# time greater than 30 days, and the second command removes all empty directories. Removed
+# files and directories are logged into /var/log which will get rotated by the system.
+
+0 2 * * * find /usr/airflow/atd-airflow/logs -type f -mtime +30 -exec rm -v {} \; \
```

Comment:
And also, how long back should we keep stuff? I went with 30 days here.

Reply:
sounds great. i can't imagine needing a longer history than that! plus the Airflow DB itself keeps a longer history, right?

Reply:
I'm not sure about the DB history TBH, but my next issue is to start backing up the DB, so I'll get a handle on what data we actually have in there when I do that and report back.